JavaScript/JavaScriptSDK/Context/Sample.ts

```diff
@@ -2,6 +2,48 @@
     "use strict";
 
     export class Sample {
-        public sampleRate: string;
+        public sampleRate: number;
+
+        public INT_MAX_VALUE: number = Math.pow(2, 32) - 1;
```

Comment:
It's too big (2^53), while I'm working with 32 bit integers (FYI - bitwise operations in javascript result with 32 bit signed integers - http://stackoverflow.com/questions/307179/what-is-javascripts-highest-integer-value-that-a-number-can-go-to-without-losin)